mailpoet/assets/js/src/automation/editor/components/workflow/step.tsx

```diff
@@ -91,6 +93,13 @@ export function Step({ step, isSelected }: Props): JSX.Element {
               : stepTypeData.title}
           </div>
         </div>
+        {error && (
+          <div className="mailpoet-automation-editor-step-footer">
+            <div className="mailpoet-automation-editor-step-error">
+              <Chip size="small">{__('Not set', 'mailpoet')}</Chip>
```

Comment:
Hmmm... `Not set`: Couldn't it be set to an invalid value as well (in theory)? Just thinking out loud, nothing to really act on I guess.

Reply:
Yep, that's how it was designed, and I think that's probably expected (@NeosinneR can confirm maybe) that when a step is invalid, we mark it as `Not set` anyway, kind of like `Not correctly set` or like `Not set well-enough to be activated, bro`. Then we'll display the error details somewhere so I guess this works.

Reply:
As Jan said 👍 this way it also simplifies the implementation.